backend/internal/controllers/router.go

```diff
@@ -43,6 +43,18 @@ func GetRouter(authenticator auth.Authenticator) chi.Router {
 
 		// "localhost:8080/books/
 		r.Get("/", bookController.GetBookList)
```

Comment:
Could you move this so we have Get("/" ... first,
and then Get("/{id}/{lang}")?
Just to keep it consistent in the ordering with the rest of the routes